src/policy/ddms.cpp

```diff
@@ -1,10 +1,23 @@
 #include <policy/ddms.h>
 
+#include <cmath>
+#include <numeric>
+
+#include <chain.h>
+#include <chainparams.h>
+#include <validation.h>
+#include <timedata.h>
 #include <consensus/tx_verify.h>
+#include <primitives/block.h>
 #include <primitives/transaction.h>
+#include <util/strencodings.h>
 
-const CScript WDMO_SCRIPT = CScript() << OP_HASH160 << std::vector<unsigned char>{11, 182, 127, 3, 232, 176, 211, 69, 45, 165, 222, 55, 211, 47, 198, 174, 240, 165, 160, 160} << OP_EQUAL; // TODO: replace with actual wdmo script; use ParseHex
 MinerLicenses minerLicenses{};
+MiningMechanism miningMechanism{};
+const CScript WDMO_SCRIPT = CScript() << OP_HASH160 << std::vector<unsigned char>{11, 182, 127, 3, 232, 176, 211, 69, 45, 165, 222, 55, 211, 47, 198, 174, 240, 165, 160, 160} << OP_EQUAL; // TODO: replace with actual wdmo script; use ParseHex
+const uint16_t MINING_ROUND_SIZE{ 100 };
+const uint32_t FIRST_MINING_ROUND_HEIGHT{ 35000 }; // TODO: change to proper value
+const uint32_t MAX_CLOSED_ROUND_TIME { MAX_FUTURE_BLOCK_TIME * 5 };
```

Comment:
Too big. I think we should put 3 times, maybe 4

Reply:
Done